vassal-app/src/main/java/VASSAL/configure/StringArrayConfigurer.java

```diff
@@ -184,6 +186,29 @@ public static String arrayToString(String[] s) {
     if (s == null || s.length == 0) {
       return "";
     }
+
```

Comment:
This is faster, if none (or few) commas happen to be in the input. And slower if the input contains many commas.